README.md

```diff
@@ -1,3 +1,150 @@
 # Progresso
 
 Minimalist and pythonic progress bar.
+
+
+## Install
+
+    pip install git+https://github.com/yohanboniface/progresso  # No pypi package yet
+
+
+## Usage
+
+    from progresso import Bar
+    bar = Bar(total=mytotalstuff)
+    for item in mystuff:
+        # do_stuff
+        bar.update()
+
+Or use `bar.iter` transparently
+
+    for item in bar.iter(mystuff):
+        do_stuff
+
+It comes with a default rendering that is enough for starting, but it's made to be
+customised very easily: just writting a template string:
+
+    bar = Bar(total=mytotalstuff, template='{prefix} {progress} ETA: {eta}')
+
+It's just plain [python formatting](https://docs.python.org/3.4/library/string.html#formatspec)
+so you can use any valid string formatting to take control over the appearance.
+For example:
+
+    bar = Bar(total=mytotalstuff, template='{progress} {percent:.2%} ETA: {eta:%H:%M:%S}')
+
+You can also just change the fill character:
+
+    bar = Bar(total=mytotalstuff, done_char='#')
+
+You can change the progress logic itself, for example to use a spinner (included):
+
+    bar = Bar(total=mytotalstuff, progress='{spinner}')
+    # 'progress' kwarg must return a valid template variable.
+    # included ones are {bar} and {spinner}
+
+You can step by more than one at a time:
+
+    for item in mystuff:
+        amount = do_stuff()
+        bar.update(step=amount)
+
+You can add more template vars by subclassing `Bar`:
```

Comment:
`ProgressBar` is probably more explicit given that `bar` is often use as foo, bar…